<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Do" name="FF Test" tests="2" failures="1" timestamp="30 Jul 2013 02:22:03 GMT" time="76.515" errors="0">
  <testcase name="Login" time="16.451" classname="com.ecalix.tests.LoginTest"/>
  <testcase name="Logout" time="10.356" classname="com.ecalix.tests.LoginTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.ecalix.tests.LoginTest.Logout(LoginTest.java:194)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)
	at java.util.concurrent.FutureTask.run(FutureTask.java:166)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- Logout -->
</testsuite> <!-- FF Test -->
